### PR TITLE
[wip] Add message property to comparators

### PR DESCRIPTION
**Pull request by lukasmartinelli**
- We can discuss about adding the `result:<comparator-name>` prefix to the message as well

### CODE REVIEW

**Comment by amishas157** [comment]
umm @lukasmartinelli ,I am not sure if I understood this properly. Why we do we want to append message? 🤔

**Comment by lukasmartinelli** [comment]
What does this need to get merged.
giphy 40

File: tests/index.js [hunk 1]
```diff
@@ -148,7 +153,11 @@ function processFixture(assert, compareFunction, fixture, callback) {
         assert.ifError(err);
         return callback(err);
       }
-      assert.deepEqual(result, fixture.expectedResult, fixture.description);
+      assert.deepEqual(
+        removeMessage(result),
```
Comment (amishas157) [inline tests/index.js]: The trick!!!

**Comment by amishas157** [comment]
Looks good to me 👌 , thanks @lukasmartinelli 🙇‍♀️

**Pull request merged**